qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -133,6 +133,11 @@ def spin2(n_electrons, n_orbitals, mapping="jordan_wigner"):
             is the number of active orbitals.
         mapping (str): Specifies the transformation to map the fermionic operator to the
             Pauli basis. Input values can be ``'jordan_wigner'`` or ``'bravyi_kitaev'``.
+        wires (Wires, list, tuple, dict): Custom wire mapping for connecting to Pennylane ansatz.
+            For types Wires/list/tuple, each item in the iterable represents a wire label
+            corresponding to the qubit number equal to its index.
+            For type dict, only int-keyed dict (for qubit-to-wire conversion) is accepted.
```

Comment:
The wires conversion in this function (and all other similar functions here) is in the OpenFermion-to-Pennylane direction, ie. ints->wires. So {int: label} makes more sense here.

In contrast to the dev/Wires classes' `wire_map`, the mapping here is operator-to-operator (or rather, ansatz-to-observable) conversion instead of operator-to-device. Here the conversion can go both ways and thus key-value pairs flip depending on the direction (which is cumbersome). This is why I ended up recommending just use a list to indicate the mapping and interpret the indices as OpenFermion ints and values as Pennylane wire labels.